index.html

```diff
@@ -588,12 +643,145 @@ <h3><dfn>MonetizationStartEventInit</dfn> dictionary</h3>
+            data-dfn-for="MonetizationStopEventInit"
+          >
+            <h3><dfn>MonetizationStopEventInit</dfn> dictionary</h3>
+            <p>
+              Dispatched once monetization has been stopped (or paused)
+            </p>
+            <pre class="idl">
+                dictionary MonetizationStopEventInit : EventInit {
+                  required DOMString paymentPointer;
+                  required DOMString requestId;
+                };
+              </pre
+            >
+            <dl>
+              <dt><dfn>paymentPointer</dfn> member</dt>
+              <dd>
+                A string representing the Payment Pointer resolved from the
+                <code>&lt;meta name=&quot;monetization&quot;&gt;</code> tag.
+              </dd>
+              <dt><dfn>requestId</dfn> member</dt>
+              <dd>
+                The Monetization ID (UUID V4) generated by the browser (see
+                [[[#flow]]]).
+              </dd>
+            </dl>
+          </section>
+        </section>
+
+        <section
+                data-dfn-for="MonetizationPendingEvent"
+        >
+          <h2><dfn>MonetizationPendingEvent</dfn> interface</h2>
+          <pre class="idl">
+            [SecureContext, Exposed=Window]
+            interface MonetizationPendingEvent : Event {
+              constructor(DOMString type, MonetizationPendingEventInit eventInitDict);
+              readonly attribute DOMString paymentPointer;
+              readonly attribute DOMString requestId;
+            };
+          </pre>
+          <section>
+            <h2>`constructor()`</h2>
+            <p>
+              The <dfn>constructor()</dfn> for {{MonetizationPendingEvent}} behaves as follows:
+            </p>
+            <ol class="algorithm">
+              <li>...TBW...</li>
+            </ol>
+          </section>
+          <section>
+            <h2><dfn>MonetizationPendingEvent/paymentPointer</dfn> attribute</h2>
+            <p>
+              When getting, returns the value it was initialized with. See
+              {{MonetizationPendingEventInit/paymentPointer}} member of
+              {{MonetizationPendingEventInit}} for more information.
+            </p>
+          </section>
+          <section>
+            <h2><dfn>MonetizationPendingEvent/requestId</dfn> attribute</h2>
+            <p>
+              When getting, returns the value it was initialized with. See
+              {{MonetizationPendingEventInit/requestId}} member of {{MonetizationPendingEventInit}} for
+              more information.
+            </p>
+          </section>
+          <section
+                  data-dfn-for="MonetizationPendingEventInit"
+          >
+            <h3><dfn>MonetizationPendingEventInit</dfn> dictionary</h3>
+            <p>
+              Dispatched once monetization has been pendingped (or paused)
+            </p>
+            <pre class="idl">
+                dictionary MonetizationPendingEventInit : EventInit {
```

Comment:
Suppose it's possible to factor out a common fields, assuming we are in consensus we want paymentPointer/requestId to contain these fields (as per current impl/explainer example code)